docs/data/external-oasis-repos.md

```diff
@@ -191,12 +191,28 @@ Once the repository is fully replicated on the OSG, the VO may proceed in publis
 If the repository ends in `.opensciencegrid.org`, the VO may ask for it to be replicated outside the US.  The
 VO should open a GGUS ticket following EGI's [PROC20](https://wiki.egi.eu/wiki/PROC20).
 
-Changing the URL of a Repository on OASIS
------------------------------------------
+Replacing an Existing OASIS Repository Server
+---------------------------------------
 
-If necessary, it is possible to change the URL of the repository server; simply have the repository administrator
-open a support ticket with the new value, and OSG operations will update OIM's OASIS repository URL for the VO.
-The GOC Stratum-1 will then be updated within an hour.
+There are two recommended ways to replace the server for an existing ``*.opensciencegrid.org`` or ``*.osgstorage.org`` repository, one without GOC intervention and one with:
```

Comment:
Why the use of double-backticks throughout this section?

Reply:
I thought I was just following prior practice in the file, but now I see that it was single backticks.  I will switch it.